modules/cram/templates/clair3_call.sh

```diff
@@ -5,9 +5,14 @@ create_bed () {
   echo -e "!{bedContent}" > "!{bed}"
 }
 
+convert_to_bam () {
```

Comment:
- add comment to indicate that this is a temporary workaround that can be removed once the issue is fixed in Clair3
- add link to the Clair3 issue